airflow/sensors/base_sensor_operator.py

```diff
@@ -58,24 +59,24 @@ class BaseSensorOperator(BaseOperator, SkipMixin):
     :type mode: str
     """
     ui_color = '#e6f1f2'
-    valid_modes = ['poke', 'reschedule']
+    valid_modes = ['poke', 'reschedule']  # type: Iterable[str]
```

Comment:
Shouldn't it be a `List[str]`?

Reply:
Good point, however I think it should be more general type because `Tuple` or ` Set` could be used here as well.